### ISSUE TITLE
Corruptive Power tactic not available

### ISSUE THREAD

**Issue by Klod91** [WarEmu/WarBugs#19779]
Can't select this tactic. It just doesnt appear in the skill tab.

![Klod_001](https://user-images.githubusercontent.com/77681824/152662277-87a71b6c-bea1-4484-a5fe-a87826cf9dce.jpg)
![Klod_002](https://user-images.githubusercontent.com/77681824/152662278-a9e68f54-a617-46fe-8903-16652e058ece.jpg)

**Comment by xpander69** [comment]
Just to make sure, did you also click the tactic from that mastery path (to obtain it)?

**Comment by Klod91** [comment]
> Just to make sure, did you also click the tactic from that mastery path (to obtain it)?

Yes. As you can see it is highlighted in yellow. So it is purchased.

**Comment by lilimx** [comment]
Hey,

want to add, that this tactic should be available with lvl 11.

**Comment by Nekkbraka** [comment]
Did you try to respec the tree? Mine is working perfectly.
![cp](https://user-images.githubusercontent.com/47920948/152683340-b32ddd75-30e2-4dec-a6b0-bd5ebb752f0b.jpg)

**Comment by lilimx** [comment]
☝️ 

Good Question!

**Comment by Rydiak** [comment]
Are you using an armor set bonus that boosts your mastery in a tree?  There seems to be a bug when using a granted mastery point to obtain a tactic.

![Lawdiak_002](https://user-images.githubusercontent.com/15963532/152922361-4b649ee8-24ad-4c46-8cd4-c83d02477775.jpg)

In this screenshot, I am using the 3-piece Challenger bonus on a Witch Hunter to add +1 to the Path of Confession mastery tree. I am then using that bonus specialization point to obtain Sanctified Bullets, but the tactic is not showing up in my list of available tactics nor is it addable to my tactic bar.

**Comment by Nekkbraka** [comment]
Try getting lvl 34 and spec this tactic again...i saw you're already lv 26.

**Comment by Rydiak** [comment]
That is likely the source of the bug then. Mastery tree tactics supposedly do not have level requirements associated with them (e.g. if you can buy them then you can use them), but there appears to be some level check behind the scenes.

**Comment by Nekkbraka** [comment]
i Just checked at Nordland wqhen it decrease 40 toons to lv 10 and i specced CP tactic..and got the same bug result. respecced at IC and its appearing again.

**Comment by Rydiak** [comment]
Thank you for testing, Nekkbraka. It is almost undoubtedly a hidden level requirement, and it isn't just this tactic either. I was able to reproduce this issue on my lowbie Witch Hunter, and I recall seeing forum posts with similar complaints to this as well for other classes.

**Comment by Rydiak** [comment]
So to confirm, there is a level requirement but it is hidden from the user? Do all masteries share the same level requirements for each tier they occupy (e.g. 3, 7, and 11 mastery levels)?

**Comment by lilimx** [comment]
Hey,

to be correct the lvl should be 23 to gain the tactic (that's the earliest lvl). The RoR Career Builder (I know, some things are wrong) said lvl 27. My low lvl Chosen said lvl 11.

Ah I will set it the label anyways.....just wanted to check what's right.

**Comment by Rydiak** [comment]
So I just went through the RoR Career Builder, and assuming Dalen pulled the data for it straight from the files, the following tactics and abilities are flagged with the wrong level requirements (or have another issue with them).

-Tactics-
Level 27 slot
Archmage-Penetrating Siphon (listed as level 1)
Chosen-Mixed Defenses (listed as level 23)
Shaman-Waagh! Frenzy (listed as level 1)

Level 23 slot
Chosen-Corruptive Power (listed as level 27)

Level 17 slot
Shaman-Pass it On (listed as level 27)
Witch Hunter-Seeker's Truth (correctly listed as level 17, but it modifies a level 25 ability; I have a post on this in the RC forum)

-Abilities-
Level 29
Marauder-Thunderous Blow (listed as level 21)

Level 25
Shaman-Sticky Feetz (listed as level 21)
White Lion-Cull the Weak (listed as level 21)
Shadow Warrior-Shadow Sting (listed as level 21)

Level 21
Squig Herder-Bounce (listed as level 20)
Marauder-Draining Swipe (listed as level 29)
White Lion-Primal Fury (listed as level 25)
White Lion-Throat Bite (listed as level 35)
Warrior Priest-Absence of Faith (listed as level 2)

**Comment by lilimx** [comment]
hmmmm, can you do a seperate report for this @Rydiak 
so it won't die here :D

**Comment by Klod91** [comment]
I hit lvl 27 and it is working now.
